index.html

```diff
@@ -1,13 +1,137 @@
+        </ul>
+
+        <nav id="navbar" class="header-body">
+            <i id="mobile-menu-close-icon" class="mobile-menu-close-icon fa-solid fa-x"></i>
+            <a class="logo" href="index.html"><img src="./assets/logo-dark.png" alt="logo"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Programe</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href=""><button class="header-btn">CC Campaign</button></a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <!-- Header Section Ends Here -->
+
+    <!-- Headline Section Starst Here -->
+    <section class="headline">
+        <h6 class="headline__greetings-heading">"Hello! Sharing world"</h6>
+        <div class="headline__main-heading">
+            <h1>Programmer Summit 2023</h1>
+            <img src="./assets/heading-bg.jpeg" alt="">
+        </div>
+        <p class="headline-description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus
+            sequi officiis incidunt odio libero a neque magnam voluptatum expedita eaque.</p>
+        <h2 class="headline-date">2023.10.15(THU) ~ 16(FRI)</h2>
+        <h6 class="headline-address">@ National Museum of Korea, Art center Nabi and more</h6>
+    </section>
+    <!-- Headline Section Ends Here -->
+
+    <!-- Program Section Starts Here -->
+    <section class="program">
+        <h3 class="program__heading">Main Program</h3>
+        <hr>
+        <ul class="program-items">
+            <li class="program-item">
+                <img src="./assets/program/lecture.png" alt="">
+                <h4>Lecture</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+            <li class="program-item">
+                <img src="./assets/program/exhibition.png" alt="">
+                <h4>CC Exhibition</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+            <li class="program-item">
+                <img src="./assets/program/chat.png" alt="">
+                <h4>Forum</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+            <li class="program-item">
+                <img src="./assets/program/workshop.png" alt="">
+                <h4>Workshop</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+            <li class="program-item">
+                <img src="./assets/program/network.png" alt="">
+                <h4>CC Ignite</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus sequi.</p>
+            </li>
+        </ul>
```

Comment:
- [ ] Kindly consider matching your design with Behance design by reducing the size of the image and font to match it

| Your design  | Behance design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/217606796-01b90298-6704-4dbe-bd9c-efb4f852953b.png)  | ![image](https://user-images.githubusercontent.com/53402600/217606865-c28e46ff-07c7-4b67-87a8-d54d4c18e4bd.png) |